xbmc/Application.cpp

```diff
@@ -323,13 +324,81 @@ extern "C" void __stdcall init_emu_environ();
 extern "C" void __stdcall update_emu_environ();
 extern "C" void __stdcall cleanup_emu_environ();
 
+namespace
+{
+bool IsPlaying(const std::string& condition,
+               const std::string& value,
+               const SettingConstPtr& setting,
+               void* data)
+{
+  return g_application.GetAppPlayer().IsPlaying();
```

Comment:
Global reference to itself?

Reply:
Just realised that myself. will off be fixed. :-)